blob/blob.go

```diff
@@ -1422,6 +1423,13 @@ type WriterOptions struct {
 	// asFunc converts its argument to driver-specific types.
 	// See https://gocloud.dev/concepts/as/ for background information.
 	BeforeWrite func(asFunc func(any) bool) error
+
+	// IfNotExist is used for conditional writes. When set to true, if
+	// a blob exists for the same key in the bucket, the write operation
```

Comment:
Please specify exactly what happens (and verify in the drivertest).

* Does the write operation succeed and just silently not write anything?
* Or does the write operation return an error? If so, what gcerrors error code does it return?

Reply:
Done!